modules/client/examples/01-client/12-prepare-uninstallation.ts

```diff
@@ -0,0 +1,78 @@
+/* MARKDOWN
+---
+title: PrepareInstallation
+---
+
+### Prepare Uninstallation of a plugin
+
+*/
+
+import {
+  Client,
+  DaoDepositSteps,
+  DepositParams,
+  GasFeeEstimation,
+  PrepareUninstallationParams,
+  PrepareUninstallationSteps,
+  SetAllowanceSteps,
+  TokenType,
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+const client: Client = new Client(context);
+
+const prepareUninstallationParams: PrepareUninstallationParams = {
+  daoAddressOrEns: "0x1234567890123456789012345678901234567890", // my-dao.dao.eth
+  versionTag: {
+    build: 1,
+    release: 1,
+  },
```

Comment:
Why do we need the version tag? Can't it be resolved from the given plugin address or from subgraph?